scripts/repoint-frontend/README.md

```diff
@@ -0,0 +1,52 @@
+# Repoint Frontend
+
+The "Repoint Frontend" script automates the process of reconfiguring the `communications-manager-api` APIs to point to a dynamic backend environment in the `comms-mgr` repository, rather than the default common backend in `internal-dev`.
+
+## Overview
+
+This script is especially useful when engineers are testing changes across both the `communications-manager-api` and `comms-mgr` repositories. By default, the `communications-manager-api` APIs point to a common backend in `internal-dev`, which is convenient when only `communications-manager-api` changes need testing. However, when changes span both `communications-manager-api` and `comms-mgr` and need testing in a specific dynamic environment (e.g. `de-todr3`), manual reconfiguration is required.
+
+The script simplifies this process by automatically:
+
+1. Pointing the APIs to the correct backend environment
+2. Disabling mTLS where necessary
+
+## Prerequisites
+
+Before running the "Repoint Frontend" script, ensure the following prerequisites are met:
+
+- You must be logged into an AWS account using SSO authentication
+- The repository must be named `communications-manager-api` to avoid potential root path issues
+
+## Steps
+
+The script performs the following steps:
+
+1. Remove mTLS if set, ensuring it is disabled if previously configured
+2. Create a new branch to contain changes to the proxy
+3. Update proxy files with the necessary configuration changes
+4. Stage, commit, and push changes to the remote repository
+5. Await DomainName availability status, with a timeout to prevent indefinite waiting
+
+## Usage
+
+```bash
+./scripts/repoint-frontend/repoint_frontend.sh <ticket ID> <environment> [options]
+```
+
+e.g.
+```bash
+./scripts/repoint-frontend/repoint_frontend.sh 0000 de-todr3
```

Comment:
I'm a little confused. Is the intention that we should run both scripts or is it an either-or situation?

Reply:
repoint_frontend.sh is the main entry point—it manages all steps and validation. The update_proxy_files.py script is called within repoint_frontend.sh to handle the line replacements in the proxy files.